src/settings/Settings.tsx

```diff
@@ -316,6 +317,16 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         supportedLevelsAreOrdered: true,
         default: false,
     },
+    "feature_invisible_crypto": {
+        isFeature: true,
+        labsGroup: LabGroup.Encryption,
+        controller: new InvisibleCryptoController(),
+        displayName: _td("labs|invisible_crypto"),
+        description: _td("labs|invisible_crypto_description"),
+        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS_WITH_CONFIG_PRIORITISED,
+        supportedLevelsAreOrdered: true,
+        default: false,
+    },
```

Comment:
Yeah, I wasn't sure how best to represent this.  Having two separate toggles for three levels seems wrong (e.g. what do we do if both "invisible" and "transition" are enabled).  I think that ideally, this would be a dropdown or something that would allow us to select one of the three levels, but I don't how to do that in settings, or if it's even possible with feature flags.  I think that if it isn't possible for feature flags, my second preference would be for the feature flag to be to enable "new cryptography modes" or some such, which would enable a selector somewhere else to select the mode.  But, basically, I didn't have time to sufficiently investigate.

Reply:
Yeah, I suspect probably a custom labs flag that's actually a drop down 9and maybe not actually a labs flag but manually put into the labs section?) That or a labs flag to enable another setting in the crypto section, although that risks people not realising they've turned the labs flag on.

Either way, would you want this PR is as it is with no option for transition mode or do we want to wait?

Reply:
see https://github.com/element-hq/element-meta/issues/2536#issuecomment-2368346133 for questions about whether we actually need a tristate

Reply:
After discussion: we do not need two settings here. But also, this whole PR needs to block on https://github.com/matrix-org/matrix-js-sdk/issues/4428.